README.md

```diff
@@ -807,6 +809,13 @@ module Api
 end
 ```
 
+If you're interested in exploring a Sinatra-based solution,
```

Comment:
This is fine but how about something simpler. :) Like: "See also: [Sinja](https://github.com/mwpastore/sinja), which extends jsonapi-serializers to provide a Sinatra-specific JSON-API framework." No need to explain your project here.